src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -88,34 +91,7 @@ public string ResourceGroupName
         // Gets or sets the storage profile.
         public StorageProfile StorageProfile { get; set; }
 
-        [JsonIgnore]
```

Comment:
Can we announce the breaking change?  Do you think it is serious?  These properties are not included in the VM response at the top level.  They are nested properties and we added these redundant properties at the top level just for pipeline.

+ @singhkays